### DISCUSSION TITLE
Any guides for making custom themes?

### DISCUSSION THREAD

**Discussion by SneakyHF** [rathena/FluxCP#364] [Q&A]
I wasn't able to find any information on this. And I saw the web support was closed on rAthena.

What software(s) do you use to create custom themes to help with the workflow?

I can't imagine it being all done in a simple IDE. Really curious about how to get started and want to make my own FluxCP themes.

**Comment by sanasol** [comment]
> I can't imagine it being all done in a simple IDE

why not?)

Actually you can update only main templates(it is about 4 files) https://github.com/rathena/FluxCP/tree/master/themes/bootstrap

And make layout and css compatible with default modules layout.

So you will have full custom skeleton/template and default inner parts thats actually doesnt have a lot of layout and can be customized one-by-one if needed.

**Comment by SneakyHF** [comment]
> > I can't imagine it being all done in a simple IDE
> 
> why not?)
> 
> Actually you can update only main templates(it is about 4 files) https://github.com/rathena/FluxCP/tree/master/themes/bootstrap
> 
> And make layout and css compatible with default modules layout.
> 
> So you will have full custom skeleton/template and default inner parts thats actually doesnt have a lot of layout and can be customized one-by-one if needed.

Do you only use one IDE? For example, I'd love for a way to see everything live as I edit. Like the blocks of the site and such. Not sure if that even is possible.

Would appreciate if you shared your workflow, just trying to learn a bit.

**Reply by farishan** [reply]
I only use Sublime text editor. Here's the result: https://rathena.org/board/files/file/4297-bootstrap-zero/

To see everything live as you edit, like the blocks of the site and such, I think that's not possible by default for now. You can add it yourself it you want to.

**Comment by farishan** [comment]
Have you checked this? https://github.com/rathena/FluxCP/blob/master/doc/user_theme.md

**Comment by SneakyHF** [comment]
Yes, I have and I'll give that IDE a try. I appreciate the help guys, thank you.

I usually work on a complete website design in Photoshop then splice up all the parts before coding, I'm just trying to understand the workflow here and what would make things easier.
